omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -113,7 +113,7 @@ public Model getCREATEModel(Representation rep) {
 		ModelImpl model = new ModelImpl()
 		        .property("encounter", new StringProperty().example("uuid"))
 		        .property("action", new StringProperty()
-		                ._enum(SwaggerSpecificationCreator.getEnumsAsList(Order.Action.class)))
+		                ._enum(EnumProperty.getEnumsAsList(Order.Action.class)))
```

Comment:
Use `new EnumProperty`, instead of `new StringProperty()._enum(EnumProperty.getEnumsAsList`

**Before:**

    new StringProperty()._enum(EnumProperty.getEnumsAsList(Order.Action.class))

**After:**

    new EnumProperty(Order.Action.class)